src/CalculatorWADTests/packages.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
```

Comment:
Can we switch to referencing these in the project files (via PackageReference) instead of using packages.config?

Are we actually using Castle.Core?

Reply:
Castle.Core is a dependency for the WinAppDriver / Appium bindings to work correctly. I've migrated over to PackageReference.